continuous-deployment/pipelines/readme.md

```diff
@@ -0,0 +1,17 @@
+# Pipelines
+
+Pipelines is a way to automatically build and test your code project and make it available to other users. It works with just about any language or project type.
+
+Pipelines combines continuous integration (CI) and continuous delivery (CD) to constantly and consistently test and build your code and ship it to any target.
+
+There are a few suggestions listed below to keep in mind when creating pipelines for continuous deployment:
+
+1. Keep your pipelines fast
```

Comment:
Nit: "As fast as feasible". Ideally I'd run a full test battery on every single build, but we all know we are conscious about trade-offs when making dev builds or smoke builds.